internal/booking/diectory/errors.go

```diff
@@ -0,0 +1,6 @@
+package diectory
+
+const (
+	ErrNoServices        = "no booking services"
```

Comment:
Would maybe be a good idea to show that its the error message and not the error itself